app/lib/relay-environment/server.ts

```diff
@@ -2,42 +2,35 @@ import {
   RelayNetworkLayer,
   urlMiddleware
 } from 'react-relay-network-modern/node8';
-import RelaySSR from 'react-relay-network-modern-ssr/node8/server';
 import {Network, Environment, RecordSource, Store} from 'relay-runtime';
 
 export default {
   initEnvironment: () => {
     const source = new RecordSource();
     const store = new Store(source);
-    const relaySSR = new RelaySSR();
 
     return {
-      relaySSR,
       environment: new Environment({
         store,
         // @ts-ignore
         network: new RelayNetworkLayer([
           urlMiddleware({
-            // TODO: set $RELAY_ENDPOINT
+            // TODOx: set $RELAY_ENDPOINT
             // url: req => process.env.RELAY_ENDPOINT
             url: _ => 'http://localhost:3004/graphql'
-          }),
-          relaySSR.getMiddleware()
+          })
         ])
       })
     };
   },
-  createEnvironment: (relayData, key) => {
```

Comment:
we're not passing any `relayData` anymore when creating the server environment

Reply:
right! I've removed it